neosnippets/java.snip

```diff
@@ -185,9 +185,15 @@ snippet main
 
 
 snippet println
+alias sout
```

Comment:
Should this be `sopl`, `soul` or `soun` if it's an abbreviation of System.out.println, compared to System.out.printf?

Reply:
this shortcut comes from IntelliJ. I am very used to it so it thought it may help also some others. Thats why I created the aliases sout/souf for this

Reply:
![Screenshot 2020-10-08 at 20 26 05](https://user-images.githubusercontent.com/34742358/95498691-7f4d7000-09a4-11eb-8289-23d2457c6180.png)

If you don't feel comfortable with this aliases i can also remove them and add them on my fork. I thought maybe some people coming from IntelliJ would try that aliases too.

Reply:
IntelliJ is common enough for Java, so I believe it's worth adding :+1: